src/libs/ajax/User.ts

```diff
@@ -3,31 +3,41 @@ import { cloneDeep, flow, unset } from 'lodash/fp';
 import { Config } from '../config';
 import axios from 'axios';
 import { getApiUrl, fetchOk, fetchAny } from '../ajax';
-
+import { DuosUserResponse, UpdateDuosUserResponse } from 'src/types/responseTypes';
+import { CreateDuosUserRequest, UpdateDuosUserRequestV1, UpdateDuosUserRequestV2 } from 'src/types/requestTypes';
+import { AcknowledgementMap, ApprovedDataset, DataSet, DuosUser, SimplifiedDuosUser } from 'src/types/model';
 
 export const User = {
-  getMe: async () => {
+  getMe: async (): Promise<DuosUserResponse> => {
     const url = `${await getApiUrl()}/api/user/me`;
     const res = await axios.get(url, Config.authOpts());
     return res.data;
   },
 
-  getById: async (id) => {
+  getById: async (id: number): Promise<DuosUserResponse> => {
     const url = `${await getApiUrl()}/api/user/${id}`;
     const res = await axios.get(url, Config.authOpts());
     return res.data;
   },
 
-  list: async (roleName) => {
+  //TODO: Determine what are legal roleNames (swagger says only these two)
```

Comment:
This is correct: https://github.com/DataBiosphere/consent/blob/9b64b3eab319c694a200f89ae2f6a62faa1efc0f/src/main/java/org/broadinstitute/consent/http/resources/UserResource.java#L75